src/main/java/uk/ac/ebi/atlas/experimentpage/tsneplot/TSnePlotDao.java

```diff
@@ -44,20 +47,22 @@ public TSnePlotDao(NamedParameterJdbcTemplate namedParameterJdbcTemplate) {
     }
 
     private static final String SELECT_T_SNE_PLOT_WITH_CLUSTERS_STATEMENT =
-			"SELECT tsne.cell_id, tsne.x, tsne.y, clusters.cluster_id" +
-					"            FROM scxa_tsne AS tsne" +
+			"SELECT coords.cell_id, coords.x, coords.y, clusters.cluster_id" +
```

Comment:
This query also has an optimisation you should switch to- see https://github.com/ebi-gene-expression-group/atlas-schemas/pull/20#issuecomment-786273695